scripts/deploy.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -e
+
+apps=(andi)
```

Comment:
This releases all the apps every time we tag the mono-repo. Is that intended to be a strategic stop-gap maneuver™ or is this the new state of things going forward?

Reply:
This is the way I intend it to work, but I need to use the strategic stop-gap maneuver™ of per-app versioning in the meantime. So I need to fix that for now.